tensorboard/webapp/webapp_data_source/BUILD

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("@npm_angular_bazel//:index.bzl", "ng_module")
+load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
```

Comment:
extraneous load.

Reply:
Done here; I'm afraid there is more of this throughout (especially unused deps).